foodsaving/stores/tests/test_feedback_api.py

```diff
@@ -0,0 +1,48 @@
+from itertools import zip_longest
+
+from dateutil.parser import parse
+from dateutil import rrule
+from dateutil.relativedelta import relativedelta
+from dateutil.rrule import rrulestr
+from django.utils.datetime_safe import datetime
+
+from django.utils import timezone
+from rest_framework import status
+from rest_framework.test import APITestCase
+from foodsaving.groups.factories import GroupFactory
+from foodsaving.stores.factories import StoreFactory, PickupDateFactory, PickupDateSeriesFactory
+from foodsaving.stores.models import PickupDate, Feedback
+from foodsaving.users.factories import UserFactory
+
+
+class TestFeedbackAPI(APITestCase):
+    """
+    This is an unit test for the Feedback API
+    """
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.now = timezone.now()
+        cls.member = UserFactory()
+        cls.group = GroupFactory(members=[cls.member, ])
+        cls.store = StoreFactory(group=cls.group)
+        cls.pickup = PickupDateFactory(store=cls.store)
+        cls.series = PickupDateSeriesFactory(max_collectors=3, store=cls.store)
```

Comment:
You won't need a series to test pickup feedback. I always like to remove code when it's not needed :) By removing cls.series, you can also get rid of cls.series.update_pickup_dates and cls.now